### ISSUE TITLE
Burner Leech and Endgame Combat error

### ISSUE THREAD

**Issue by JonScarlet** [ReikaKalseki/Reika_FactorioMods_Issues#230]
I admit I'm not sure how this happened considering I'm past using burner inserters, but this occurred during my run.

[![Screenshot from Gyazo](https://gyazo.com/4e3f4447ec09f39a57efc890a511847d/raw)](https://gyazo.com/4e3f4447ec09f39a57efc890a511847d)

I know I was in the process of laying down some turrets (with the Burner Leech mod enabled) when it happened, although I don't know what specific action caused the error, although I do believe it's the first time the turrets would've attacked since I got Turret Range 1 in this playthrough

Both this and Burner Leech did it, so I've posted this on that one's mod portal.

**Comment by JonScarlet** [comment]
Tried to start from my save file just before writing this. I got the same error when I tried to place a gun turret down and did nothing else between loading the file and doing that action.

As far as research goes for the turrets, I only had bullet speed and dmg 2 and turret range 1. I did go back to a previous autosave that didn't have range 1 researched, and I was able to place one down. However, after I got the upgrade, it appeared again.

I don't know if it applies to any of the other range upgrades sine this is a brand-new run for me, nor if burner leech actually has any connection to it, but there is something with getting range 1 and placing down new turrets.
